### ISSUE TITLE
Add WangscapeGui to Windows binary package

### ISSUE THREAD

**Issue by serin-delaunay** [Wangscape/Wangscape#173]
AppVeyor builds WangscapeGui successfully, but doesn't include the resulting .exe in the binary package.
- [ ] Include WangscapeGui.exe in Wangscape folder.
- [ ] Include all required Qt DLLs.
- [ ] Add Qt license to licenses folder.
- [ ] Update instructions for building on Windows (install Qt, make sure `uic` and `moc` are in `%PATH%`).

**Comment by V1shvesh** [comment]
@serin-delaunay Hello again! Can I work upon this issue?

**Comment by serin-delaunay** [comment]
@V1shvesh Sure! Let me know if you have any questions :)

**Comment by V1shvesh** [comment]
Again, I'm a lvl0 n00b. So bear with me as I keep asking silly questions.

**Comment by V1shvesh** [comment]
Should I include the whole Releases folder in Wangscape? Or only the .exe file? Also, do I use `robocopy`?

**Comment by serin-delaunay** [comment]
No problem! The `Release` folder will have a lot of junk generated by Visual Studio, so don't copy the whole folder - just enough to make the GUI application run. I think the files listed above should suffice. `robocopy` has nice syntax for copying a list of files between two directories, so it makes sense to use it in this case. Single files can use `copy` or `xcopy`.

**Comment by V1shvesh** [comment]
@serin-delaunay And this is to be done after build, right?

**Comment by serin-delaunay** [comment]
Yes, in the [`after_build`](https://github.com/Wangscape/Wangscape/blob/master/appveyor.yml#L56) section of `appveyor.yml`.

**Comment by V1shvesh** [comment]
Cool. 

One more thing. I don't know how I can get the file structure being created here. How will I get to know what's in Releases folder, where is it etc.?

**Comment by serin-delaunay** [comment]
I would add a `dir /s /b` command (or [powershell equivalent](https://stackoverflow.com/questions/1479663/how-do-i-do-dir-s-b-in-powershell)) to the script, and examine the log output. It should be removed or commented out before merging.

**Comment by V1shvesh** [comment]
But doesn't appveyor run only during pull requests?

**Comment by serin-delaunay** [comment]
You can open a pull request early and mark it `WIP` to say it's not ready for review/merge, or you can make an account on AppVeyor and set it to build your fork whenever you push to a branch.
